src/multer/index.ts

```diff
@@ -0,0 +1,28 @@
+import multer from 'multer';
+import { AppError } from '../modules/errors/AppError.ts';
+import { Request } from 'express';
+
+const multerStorage = multer.diskStorage({
+  destination: (req, file, cb) => {
+    cb(null, 'public/img/todos');
+  },
+  filename: (req, file, cb) => {
+    // todo-67676bsdad-33444.jpeg
+    const ext = file.mimetype.split('/')[1];
+    //@ts-ignore
```

Comment:
It's better to apply specific ignore annotations rather than a general one so it's more clear what kind of issue we want to ignore.

Reply:
@joseph-dev  I have this filter so it decline all not image files
<img width="718" alt="Screenshot 2024-01-14 at 18 05 03" src="https://github.com/ArtyomZayarny/todo-api/assets/15728688/3bd8d7b7-a325-4f04-b126-a411008bbe39">

Reply:
It's not about images... I was referring to TypeScript annotations e.g. `@ts-ignore`, this is a general one which doesn't give any details what kind of error you're ignoring here.